<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHChooseGrabberType</class>
 <widget class="QDialog" name="QHChooseGrabberType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>154</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Choose Grabber Type</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose Grabber Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typecomboBox">
       <item>
        <property name="text">
         <string notr="true">R2R3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">R3X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">R5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">R5X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">R6(C-PHY)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okpushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelpushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
